lib/internal/per_context/primordials.js

```diff
@@ -63,10 +86,16 @@ function copyPropsRenamedBound(src, dest, prefix) {
     if ('get' in desc) {
       copyAccessor(dest, prefix, newKey, desc);
     } else {
-      if (typeof desc.value === 'function') {
-        desc.value = desc.value.bind(src);
+      const { value } = desc;
+      if (typeof value === 'function') {
+        desc.value = value.bind(src);
```

Comment:
unrelated change?

Reply:
This preserves the original `value`, so that it can be `applyBind`ed later.

Reply:
But you `applyBind` `desc.value`, not `value` 🤔

Reply:
Right, I forgot to fix that after using <kbd>Ctrl+C</kbd> and <kbd>Ctrl+V</kbd>.